applications/FluidDynamicsApplication/custom_processes/embedded_skin_visualization_process.cpp

```diff
@@ -34,101 +35,165 @@ namespace Kratos
 {
 
 /* Public functions *******************************************************/
+Parameters EmbeddedSkinVisualizationProcess::GetDefaultSettings()
+{
+    Parameters default_settings(R"(
+    {
+        "model_part_name"                     : "",
+        "visualization_model_part_name"       : "EmbeddedSkinVisualizationModelPart",
+        "shape_functions"                     : "",
+        "reform_model_part_at_each_time_step" : false,
```

Comment:
I would call this sth with „Update“
„Reform“ is usually associated with DOF-things

Reply:
I'd rather keep `reform`. Indeed, this was my original idea, as from the output perspective, what comes to our minds when using `reform_` is `MultipleFiles` flag in the `GiDOutput`, which is of course required in this case.